CODEOWNERS

```
@@ -5,4 +5,4 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
-* @fmvilas @derberg @github-actions[bot]
\ No newline at end of file
+* @fmvilas @derberg @dalelane @github-actions[bot]
```

```suggestion
* @fmvilas @derberg @dalelane @asyncapi-bot-eve
```


related https://github.com/asyncapi/.github/pull/139